dxr/plugins/python/indexers.py

```diff
@@ -25,14 +26,28 @@
 }
 
 
+class _FileToIgnore(object):
+    """A file that we don't want to bother indexing, usually due to
+    syntax errors.
+
+    """
+    def is_interesting(self):
+        return False
+FILE_TO_IGNORE = _FileToIgnore()
+
+
+
 class TreeToIndex(TreeToIndexBase):
     def __init__(self, *args, **kwargs):
         super(TreeToIndex, self).__init__(*args, **kwargs)
 
         self.python_path = getattr(self.tree, 'plugin_python_path', self.tree.source_folder)
```

Comment:
We should change this to use the new config system (config_schema, self.plugin_config(), etc.) at some point. I thought we had....